src/redux/categories/categories.js

```diff
@@ -0,0 +1,3 @@
+const Categories = () => <h1>Under Construction</h1>;
+
+export default Categories;
```

Comment:
- [ ] As mentioned above all components should be in the `components` folder and responsible for rendering data. The `redux` folder is used for component state management and therefore should not host the component itself. Please move this component to the component folder